include/onnxruntime/core/session/experimental_onnxruntime_cxx_api.h

```diff
@@ -47,6 +47,19 @@ struct Session : Ort::Session {
   std::vector<std::vector<int64_t> > GetOverridableInitializerShapes() const;
 };
 
+struct Value : Ort::Value {
+  Value(OrtValue* p)
+      : Ort::Value(p){};
+
+  template <typename T>
+  static Ort::Value CreateTensor(T* p_data, size_t p_data_element_count, const std::vector<int64_t>& shape);
```

Comment:
Also, these APIs use the AllocatorWithDefaultOptions by default. Either we should document this or change the name of these APIs to reflect this assumption.

Reply:
I put a note in the header section of the file. I don't know much about how `allocator` is used internally so if there's a better default setting, let me know. I assumed by the name, it was the best generic setting to use.